test/check/check.go

```diff
@@ -66,13 +66,13 @@ func CheckConfigCondition(conditionType ConditionType, conditionStatus Condition
 	}
 
 	if timeout != CheckImmediately {
-		Eventually(getAndCheckCondition, timeout, time.Second).ShouldNot(HaveOccurred(), fmt.Sprintf("Timed out waiting for the condition, current config:\n%v", configToYaml(config)))
+		Eventually(getAndCheckCondition, timeout, 10*time.Millisecond).ShouldNot(HaveOccurred(), fmt.Sprintf("Timed out waiting for the condition, current config:\n%v", configToYaml(config)))
```

Comment:
can this be a constant?

Reply:
It is used only twice. I personally prefer having these kind of values inline, directly on the place where they are needed. Otherwise we would pile up all time constants at the top of the file, I find that more confusing and harder to read, with no notable benefit.